applications/ConstitutiveLawsApplication/custom_constitutive/generic_small_strain_high_cycle_fatigue_law.h

```diff
@@ -154,6 +154,12 @@ class KRATOS_API(CONSTITUTIVE_LAWS_APPLICATION) GenericSmallStrainHighCycleFatig
     ///@name Operations
     ///@{
 
+    /**
+     * @brief Initialize the material response in terms of Cauchy stresses
+     * @see Parameters
+     */
+    void InitializeMaterialResponseCauchy (ConstitutiveLaw::Parameters& rValues) override;
```

Comment:
I think that, in order to be safe if using TL or UL element, you should implement the `InitializeMaterialResponseKirchoff` etc... and calling by default the 'InitializeMaterialResponseCauchy ' right @loumalouomega ?

Reply:
I think so

Reply:
You are right. Done!